### ISSUE TITLE
Completion not working at the beginning of a line

### ISSUE THREAD

**Issue by Libavius** [satamas/fortran-plugin#33]
Thanks again for adding code completion, in many cases, it works like a charm!

However, code completion does for me not trigger at the beginning of a line, if no call or write statement precedes the name of the variable/function. (If the variable is given to a function as an argument, the completion works)

thanks a lot for your work!

**Comment by satamas** [comment]
Yes, I know about this issue. Thing is that I've added reference completion and at the beginning of the line there is no reference for now due to complexities of our parser.

**Comment by satamas** [comment]
There is still no reference completion at the beginning of the line, but I've added keyword completion there
